client/src/components/video.vue

```diff
@@ -401,6 +404,18 @@
       }
     }
 
+    onBlur() {
+      if (!this.focused || !this.hosting || this.locked) {
```

Comment:
shouldn't this be `this.focused` vs `!this.focused`? did you test if it works?

Reply:
Tested. It works in Chrome & Firefox, appearently it is called in right order. According to docs, it seems it fires before `onMouseLeave` that sets `focused` to `false`.

> The [focusout ](https://developer.mozilla.org/en-US/docs/Web/API/Element/focusout_event) event fires when an element **is about to lose focus**. The main difference between this event and blur is that focusout bubbles while blur does not.


It should only send those keys, when cursor is in browser - so browser is focused and listenting for keys. It sould not send keys, when I am e.g. with cursor in chat, but have active keyboard and I press `Alt+Tab`.

Reply:
awesome looks good to me then